RadarSDK/RadarLocationManager.m

```diff
@@ -727,7 +727,7 @@ - (void)handleLocation:(CLLocation *)location source:(RadarLocationSource)source
                 duration = -[location.timestamp timeIntervalSinceNow];
             }
             BOOL arrival = source == RadarLocationSourceVisitArrival;
-            stopped = (distance <= options.stopDistance && duration >= options.stopDuration) || arrival;
+            stopped = (distance <= options.stopDistance && duration >= options.stopDuration);
```

Comment:
Removed stop detection using arrival, can feature flag this too if needed

Reply:
I think we should feature flag such that if it's default (3000 & 3000) we do count arrivals as stop (old impl) and if it's non default we don't.